manifest/oci.go

```diff
@@ -241,3 +251,34 @@ func (m *OCI1) ImageID([]digest.Digest) (string, error) {
 	}
 	return m.Config.Digest.Hex(), nil
 }
+
+// IsEncryptedLayer indicates whether the blob is is encrypted
+func IsEncryptedLayer(b types.BlobInfo) bool {
```

Comment:
Does this _have_ to be a public function like this, with knowledge about all possible kinds of manifest types?

Maybe it should be something the image/manifest implementation returns (in _some_ way), with each format implementation only dealing with its own formats.

Reply:
This can be private - ditto for the other 2 functions `GetEnc/DecMediatype`